src/backend/db/users.js

```diff
@@ -8,11 +8,11 @@ import { formatDate } from "../utils/authUtils";
 export const users = [
   {
     _id: uuid(),
-    firstName: "Adarsh",
-    lastName: "Balika",
-    username: "adarshbalika",
-    password: "adarshBalika123",
+    firstName: 'Adarsh',
+    lastName: 'Balika',
+    username: 'adarshbalika',
+    password: 'adarshBalika123',
```

Comment:
change the dummy username to something else. (if you remember). Better do it in the early stages of building the app itself.